src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Completion.cs

```diff
@@ -58,7 +58,7 @@ public class Command
         /// <summary>
         /// The identifier of the actual command handler, like `vsintellicode.completionItemSelected`.
         /// </summary>
-        public string CommandStr { get; set; }
+        public string command { get; set; }
```

Comment:
Casing shouldn't matter so let's have this be capitalized in keeping with C# styling.

Reply:
Because the class name is also Command, I had to use command for the field to avoid naming conflict.